api/tools/bots/join.ts

```
@@ -1,58 +1,245 @@
+          response?: {
+            status: number;
+            data: any;
+          };
+        };
+
+        if (error instanceof Error) {
+          errorMessage += error.message;
+
+          // Check for axios error with response data using type assertion
+          const axiosError = error as AxiosErrorWithResponse;
+          if (axiosError.response && axiosError.response.data) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)